api/src/utils/github.js

```diff
@@ -58,11 +58,33 @@ export const getUserByName = async (username) => {
   log.debug(`getUserByName ${username}`)
   const installations = await getAuthenticatedApps()
   const config = getConfig()
-
   const response = await installations.apps[
-    config.primaryOrg
+    config.primaryOrg.toLowerCase()
   ].authenticatedRequest('GET /users/{username}', {
     username,
   })
   return response.data
 }
+
+export const getRequestStatuses = async (requests) => {
```

Comment:
i would like to see this function tested

Reply:
Is the scope of this great enough to be its own sprint goal? I've started writing a test for this, but it will take me a while to figure out the jest testing framework.